### DISCUSSION TITLE
improvement on balance system

### DISCUSSION THREAD

**Discussion by TeomanOFF** [CatimaLoyalty/Android#1905] [Ideas]
The cart icon here 
![Screenshot_20240526-090549_Catima](https://github.com/CatimaLoyalty/Android/assets/170851738/b9fcc81f-571b-4039-a327-047031842441)
should have a toggle switch here 
![Screenshot_20240526-090540_Catima](https://github.com/CatimaLoyalty/Android/assets/170851738/aacc53df-9e28-46f4-acb8-5529c81279a7)
 to decide if this card uses balance system
and and if it's enabled then it shows the balance unit chooser that should also have a option to make a custom unit and the set balance box should be removed because it will be accessed from here 
![Screenshot_20240526-090549_Catima](https://github.com/CatimaLoyalty/Android/assets/170851738/a1e085b5-af8b-437e-a3ec-41603108e9c7)
- *Teoman* **Catima User**

**Comment by TheLastProject** [comment]
I don't really understand your feature request and, more importantly, I'm not seeing what actual issue it would solve. Keeping Catima simple is a design goal so changes, especially those that add new options, need clear use cases so that they actually solve real-life problems instead of just make the app more complex.

1. Toggle switch for cart icon to enable or disable

What problem does this solve? It's one very small icon, there is nothing else using that UI space. It seems you're asking for an extra option to hide one very small button when hiding it won't open up any new screen estate? Why?

2. have a option to make a custom unit

What's the use case of this? Catima uses the ISO 4217 standard (and "Points") which means supporting hundreds of currencies used around the whole world already: https://en.wikipedia.org/wiki/ISO_4217#List_of_ISO_4217_currency_codes.

**Reply by TeomanOFF** [reply]
No im saying not to disable cart but to have the cart icon even its set to 0 for quick access and for the unit part a custom choice will be good for like other than currency or point things, exemples: For library card to have the custom thing to write Like borrowed books and if you have 0 borrowed and when you borrow one a quick add instead of going in edit tab 2 then editing it's much quicker to click the cart
